recipes/percolator/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: percolator
+  version: '3.0'
+
+source:
+  git_url: https://github.com/percolator/percolator.git
+  git_rev: c8a48c4 
+
+build:
+  number: 1
+  skip: True # [osx]
```

Comment:
you probably want to add a build-string to your package that indicates the BOOST version?